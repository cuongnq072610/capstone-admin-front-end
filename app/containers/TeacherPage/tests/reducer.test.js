import { fromJS } from 'immutable';
import teacherPageReducer from '../reducer';

describe('teacherPageReducer', () => {
  it('returns the initial state', () => {
    expect(teacherPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
