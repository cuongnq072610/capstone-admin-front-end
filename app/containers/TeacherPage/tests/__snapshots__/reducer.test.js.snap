// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teacherPageReducer returns the initial state 1`] = `
Immutable.Map {
  "teachers": Immutable.List [],
  "departments": Immutable.List [],
  "isLoadingUpdate": false,
  "isLoading": false,
  "errors": "",
  "message": "",
  "courses": Immutable.List [],
  "isLoadingCourse": false,
}
`;
